lib/rake/build.rake

```diff
@@ -16,14 +30,10 @@ namespace :build do
       # Workaround for https://github.com/karma-runner/karma-phantomjs-launcher/issues/120
       RakeUtils.npm_rebuild 'phantomjs-prebuilt'
 
-      if rack_env?(:staging)
-        ChatClient.log 'Updating <b>apps</b> i18n strings...'
-        RakeUtils.system './sync-apps.sh'
-      end
-
       ChatClient.log 'Building <b>apps</b>...'
       npm_target = (rack_env?(:development) || ENV['CI']) ? 'build' : 'build:dist'
       RakeUtils.system "npm run #{npm_target}"
+      File.write(apps_dir('commit_hash'), RakeUtils.git_folder_hash(apps_dir))
```

Comment:
Does this interfere with [the `commit_hash` file the apps packager writes?](https://github.com/code-dot-org/code-dot-org/blob/2f7d0a08f836ef1268aeec0964764d75268b42e6/lib/cdo/aws/s3_packaging.rb#L93-L93)